<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeConvertWidget</class>
 <widget class="QWidget" name="TimeConvertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时间转换</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_UnixTimestamp">
       <property name="text">
        <string>当前Unix时间戳（Unix timestamp）：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_UnixTimestamp"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbtn_UnixTimestampCopy">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_InputTimestamp">
       <property name="text">
        <string>转换Unix时间戳（Unix timestamp）：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_InputTimestamp">
       <property name="whatsThis">
        <string/>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_InputTimestampCopy">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbtn_Unix2Normal">
         <property name="text">
          <string>转换↓</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbtn_Normal2Unix">
         <property name="text">
          <string>↑转换</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboBox_Format">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_NormalTime"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pbtn_NormalTimeCopy">
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_histroy"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
